test/python/algorithms/state_fidelities/test_compute_uncompute.py

```diff
@@ -56,7 +56,22 @@ def test_1param_pair(self):
         """test for fidelity with one pair of parameters"""
         fidelity = ComputeUncompute(self._sampler)
         job = fidelity.run(
-            self._circuit[0], self._circuit[1], self._left_params[0], self._right_params[0]
+            self._circuit[0],
+            self._circuit[1],
+            self._left_params[0],
+            self._right_params[0],
+        )
+        result = job.result()
+        np.testing.assert_allclose(result.fidelities, np.array([1.0]))
```

Comment:
It would be nice to test some additional values which check the "local" feature. E.g. the overlap of the `plus` and `zero` circuits would differ from local to global fidelity, right?